src/ontodia/workspace/workspace.ts

```diff
@@ -123,15 +133,23 @@ export class Workspace extends Component<WorkspaceProps, State> {
                 selectedLanguage: this.diagram.getLanguage(),
                 onChangeLanguage: this.changeLanguage,
                 onShowTutorial: this.showTutorial,
-                isViewOnly,
+                hidePanels,
+                isLeftPanelOpen: this.state.isLeftPanelOpen,
+                onLeftPanelToggle: () => {
+                    this.setState(prevState => ({isLeftPanelOpen: !prevState.isLeftPanelOpen}));
+                },
+                isRightPanelOpen: this.state.isRightPanelOpen,
+                onRightPanelToggle: () => {
+                    this.setState(prevState => ({isRightPanelOpen: !prevState.isRightPanelOpen}));
+                },
             }),
         } as MarkupProps & React.ClassAttributes<WorkspaceMarkup>);
     }
 
     componentDidMount() {
         this.diagram.initializePaperComponents();
 
-        if (this.props.isViewOnly) { return; }
+        if (this.props.hidePanels) { return; }
```

Comment:
We could safely remove this `if` statement. (I think `showTutorialIfNotSeen()` call should be removed as well.)